test/test_js.expected

```diff
@@ -0,0 +1,10 @@
+(module
+ (type $i32_i32_=>_i32 (func (param i32 i32) (result i32)))
+ (func $adder (param $0 i32) (param $1 i32) (result i32)
+  (i32.add
+   (local.get $0)
+   (local.get $1)
+  )
+ )
+)
+OK!
```

Comment:
This can't be generated because the JS emits sync, so the OK is at the end, but in native, it prints first. Weird.

Reply:
FWIW I'm totally fine with dropping that print line from the test.

Reply:
Cool, I'll remove that